lightning/src/ln/peer_handler.rs

```diff
@@ -553,7 +554,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	/// [`socket_disconnected()`].
 	///
 	/// [`socket_disconnected()`]: PeerManager::socket_disconnected
```

Comment:
I think we should document the new parameters on these pub methods. Also in both, rename: `s/net_address/remote_network_address`

Reply:
Should I change `their_net_address` inside the `Peer` struct to `their_remote_network_address` as well?